content/Lab1/40_Deploy_Test_Machines/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "Deploy Test Machines"
-weight: 60
+weight: 40
 ---
 
 In this section, we will deploy test machines to test traffic flow to/from the firewall. We will first describe and walk through the relevant terraform code snippets and then proceed with the deployment.
 
-[Test Machines Setup - Description](../60_Deploy_Test_Machines/1_Test_machines_setup.md)
+[Test Machines Setup - Description](../40_Deploy_Test_Machines/1_Test_machines_setup.md)
```

Comment:
please rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description